python/py-catalogue/Portfile

```diff
@@ -6,11 +6,12 @@ PortGroup           python 1.0
 
 epoch               1
 # Don't upgrade past py-spaCy's highest supported version.
-github.setup        explosion catalogue 1.0.0 v
+github.setup        explosion catalogue 2.0.1 v
```

Comment:
this version is above the one supported by the py-spaCy's port in MacPorts

Reply:
Thanks done.